packages/web-api/.gitignore

```diff
@@ -1,5 +1,6 @@
 # node / npm stuff
 /node_modules
+package-lock.json
```

Comment:
Would you mind reverting this change? We'll do this in a different pull request.
```suggestion
```